Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -23,10 +23,8 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
-{
-    constructor(delegate)
-    {
+WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel {
+    constructor(delegate) {
```

Comment:
we dont make changes like this unless there's something relevant to modify, so please undo this change